### PR TITLE
Add entity scope to documentation

### PR DESCRIPTION
**Pull request by mohammadmuazam**

### CODE REVIEW

File: docs/resources/team.md [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
 subcategory: ""
 description: |-
-  Use this resource to manage the Team meta details like Name, descripton etc.
+  Use this resource to manage the Team meta details like Name, description etc. The name of the Team must be unique within an organization.
 ---
 
 # squadcast_team (Resource)
 
-Use this resource to manage the Team meta details like Name, descripton etc.
+Use this resource to manage the Team meta details like Name, description etc. The name of the Team must be unique within an organization.
```
Comment (Lucifer8729) [inline docs/resources/team.md:11]: I thought we are not using `organization` keyword

File: internal/provider/resource_team_meta.go [hunk 2]
```diff
@@ -13,7 +13,7 @@ import (
 
 func resourceTeam() *schema.Resource {
 	return &schema.Resource{
-		Description: "Use this resource to manage the Team meta details like Name, descripton etc.",
+		Description: "Use this resource to manage the Team meta details like Name, description etc. The name of the Team must be unique within an organization.",
```
Comment (Lucifer8729) [inline internal/provider/resource_team_meta.go:16]: same question

**Review by Lucifer8729 — APPROVED** [review]
rest looks good

File: docs/resources/squad.md [hunk 3]
```diff
@@ -3,12 +3,12 @@
 page_title: "squadcast_squad Resource - terraform-provider-squadcast"
 subcategory: ""
 description: |-
-  Squads https://support.squadcast.com/docs/squads are smaller groups of members within Teams. Squads could correspond to groups of people that are responsible for specific projects within a Team.
+  Squads https://support.squadcast.com/docs/squads are smaller groups of members within Teams. Squads could correspond to groups of people that are responsible for specific projects within a Team. The name of the Escalation Policy must be unique within a Team.
```
Comment (Dineshsq) [inline docs/resources/squad.md]: The name of the Squad must be unique within a Team

File: docs/resources/squad.md [hunk 4]
```diff
@@ -3,12 +3,12 @@
...
 ---
 
 # squadcast_squad (Resource)
 
-[Squads](https://support.squadcast.com/docs/squads) are smaller groups of members within Teams. Squads could correspond to groups of people that are responsible for specific projects within a Team.
+[Squads](https://support.squadcast.com/docs/squads) are smaller groups of members within Teams. Squads could correspond to groups of people that are responsible for specific projects within a Team. The name of the Escalation Policy must be unique within a Team.
```
Comment (Dineshsq) [inline docs/resources/squad.md]: The name of the Squad must be unique within a Team

File: internal/provider/resource_squad.go [hunk 5]
```diff
@@ -15,7 +15,7 @@ import (
 
 func resourceSquad() *schema.Resource {
 	return &schema.Resource{
-		Description: "[Squads](https://support.squadcast.com/docs/squads) are smaller groups of members within Teams. Squads could correspond to groups of people that are responsible for specific projects within a Team.",
+		Description: "[Squads](https://support.squadcast.com/docs/squads) are smaller groups of members within Teams. Squads could correspond to groups of people that are responsible for specific projects within a Team. The name of the Escalation Policy must be unique within a Team.",
```
Comment (Dineshsq) [inline internal/provider/resource_squad.go]: The name of the Squad must be unique within a Team

**Review by Dineshsq — CHANGES_REQUESTED** [review]

**Review by roshan8 — APPROVED** [review]

File: internal/provider/resource_team_meta.go (hunk 2 above)
Comment (mohammadmuazam) [reply internal/provider/resource_team_meta.go:16]: we cannot use "teams" for these

File: docs/resources/team.md (hunk 1 above)
Comment (mohammadmuazam) [reply docs/resources/team.md:11]: we cannot use "teams" for these

**Review by Dineshsq — APPROVED** [review]

**Pull request merged**